### ISSUE TITLE
prepack fails when a switch statement has duplicate expressions in case clauses

### ISSUE THREAD

**Issue by a8m** [facebook/prepack#1645]
Code example:
```js
switch (Math.floor(Math.random() * 10)) {
  case 1:console.log('1'); break;
  case 1:console.log('1'); break;
  default: console.log('other');
}
```

Output:
```
Invariant Violation: pushing false
This is likely a bug in Prepack, not your code. Feel free to open an issue on GitHub.
....
```

[Link to prepack-repl](https://prepack.io/repl.html#M4dwlgLgxgFgBACgLIEMIwHQDMA2B7PAJ2TU0JQDsATPAWwQEo4AqOARgAYGmBvAKDhwoKYAFN2ALih4KwPDlEZ8AcwQByNmoYBuOACNColAGttAoSPFspMuQqV5VGrboNHT5qqKwoArjggJIVt5RRV1PHRRQhc+AF8+IA)

**Comment by NTillmann** [comment]
To address this...
1. A test in test/serializer/abstract should be created, similar to other tests in that directory.
2. The root cause of the issue needs to be addressed.

**Comment by a8m** [comment]
Closed via #1681
